application/common/models/Mfa.php

```diff
@@ -148,7 +148,21 @@ public function afterSave($insert, $changedAttributes)
     public function beforeDelete()
     {
         $this->clearFailedAttempts('when deleting mfa record');
-        
+
+        // first delete the webauthn children to avoid a foreign key constraint error
+        foreach ($this->mfaWebauthns as $child) {
```

Comment:
I considered putting this in an if block that checked that the `mfa` was of type webauthn, but I figured none of the other types would have any mfaWebauthn children. So, the `foreach` loop would just be skipped anyway.